server/sonar-server/src/main/java/org/sonar/server/computation/source/LastCommitVisitor.java

```diff
@@ -35,12 +36,16 @@
 
 public class LastCommitVisitor extends PathAwareVisitorAdapter<LastCommitVisitor.LastCommit> {
 
+  private static final long MILLISECONDS_PER_DAY = 1000 * 60 * 60 * 24;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Cast one of the operands of this multiplication operation to a "long". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS2184)